src/lib/core/WeaveMessageLayer.h

```diff
@@ -792,6 +792,30 @@ inline bool WeaveMessageLayer::IsBoundToLocalIPv6Address(void) const
 #endif
 }
 
+/**
+ * Bit field definitions for IEEE EUI-64 Identifiers.
+ */
+enum
+{
+    kEUI64_UL_Mask          = 0x0200000000000000ULL,        /**< Bitmask for the Universal/Local (U/L) bit within an EUI-64 identifier.
+                                                                 A value of 0 indicates the id is Universally (globally) administered.
+                                                                 A value of 1 indicates the id is Locally administered. */
+
+    kEUI64_UL_Unversal      = 0,                            /**< Universal/Local bit value indicating a Universally administered EUI-64 identifier. */
+
+    kEUI64_UL_Local         = kEUI64_UL_Mask,               /**< Universal/Local bit value indicating a Locally administered EUI-64 identifier. */
+
+    kEUI64_UL_MaxTestLocal  = 65535,                        /**< EUI-64 identifiers less or equal than this value are considered local for testing convenience. */
```

Comment:
I think it would be better to define this in terms of a special range of Weave Node Ids, rather than EUI-64s.  So maybe make this its own enum with a name like kMaxAlwaysLocalWeaveNodeId.

Reply:
Done.